modules/webgl/test/context/state-tracker/context-state.spec.ts

```diff
@@ -469,7 +485,9 @@ test('WebGLState#withGLParameters framebuffer', t => {
   t.end();
 });
 
-test('WebGLState#withGLParameters empty parameters object', t => {
+test('WebGLState#withGLParameters empty parameters object', async t => {
+  const webglDevice = await getWebGLTestDevice();
```

Comment:
A lot of repetition here. Would it be worth defining a helper function `webGlTest()` that passed in the created context?

```
webGlTest('WebGLState#withGLParameters empty parameters object', async (t, webglDevice) => {...})
```

Reply:
> A lot of repetition here. Would it be worth defining a helper function `webGlTest()` that passed in the created context?
> 
> ```
> webGlTest('WebGLState#withGLParameters empty parameters object', async (t, webglDevice) => {...})
> ```

That could be a good idea.
- We'll be moving most of the WebGL specific tests (not this file of course) to multi-platform, so we will want to call `getTestDevices()` instead of `getWebGLTestDevice()`, but I suppose we could supply a `testDevices` param.
- I'd like to move to jest and I wonder if there is not a `beforeTest()` type method that could wait for the device and store it in a global var.
- I tend to be a bit wary of wrapping things, it makes the code base a bit less clear.